indico/web/forms/fields/principals.py

```diff
@@ -7,13 +7,16 @@
 
 from __future__ import absolute_import, unicode_literals
 
+import itertools
 import json
 from operator import attrgetter
 
+from sqlalchemy.orm import joinedload
 from wtforms import HiddenField
 
 from indico.core.db.sqlalchemy.principals import PrincipalType, serialize_email_principal
 from indico.core.permissions import get_permissions_info
+from indico.modules.categories.models.categories import Category
 from indico.modules.events import Event
 from indico.modules.events.contributions.models.contributions import Contribution
 from indico.modules.events.roles.util import serialize_role
```

Comment:
I'd move this one inside the function as well... just a matter of time before it causes import issues as well 🙈 

And maybe rename it to `serialize_event_role`...